article/app/views/liveblog/liveBlogBodyContentAMP.scala.html

```diff
@@ -13,7 +13,7 @@
         @* Add advert every 5 blog posts, up to a maximum of 8 *@
         @if(blockGroup.length == 5 && index < 8) {
             <div id="amp-ad-@index" data-sort-time="1" class="block amp-ad-container amp-ad-container--live-blog">
```

Comment:
I know this is bigger than this PR, but do you think at some point we can unify the two amp ad templates that seem to have come about? Seems silly to have two things so similar.